r/NEWS.md

```diff
@@ -31,7 +31,8 @@
 * `map_batches()` returns a `RecordBatchReader` and requires that the function it maps returns something coercible to a `RecordBatch` through the `as_record_batch()` S3 function. It can also run in streaming fashion if passed `.lazy = TRUE`. (ARROW-15271, ARROW-16703)
 * Functions can be called with package namespace prefixes (e.g. `stringr::`, `lubridate::`) within queries. For example, `stringr::str_length` will now dispatch to the same kernel as `str_length`. (ARROW-14575)
 * Support for new functions:
-  * `lubridate::parse_date_time()` datetime parser: (ARROW-14848, ARROW-16407)
+  * `lubridate::parse_date_time()` datetime parser: (ARROW-14848, ARROW-16407, ARROW-16653)
+    * most formats are now supported.
```

Comment:
Are there any formats we still do not support? With the exception of locale-specific ones on Windows?

Reply:
`strftime` supports: `%a %A %w %d %b %B %m %y %Y %H %I %p %M %z %Z %j %U %W %x %X %% %G %V %u`
`strptime` on Windows supports: `"%d", "%H", "%j", "%m", "%T", "%S", "%q", "%M", "%U", "%w", "%W", "%y", "%Y", "%R", "%T"`
`strptime` on non-Windows in additionally supports: `"%a", "%A", "%b", "%B", "%Om", "%OS", "%I", "%p", "%r", "%z"`
`parse_date_time` supports on Windows: `"H", "I", "j", "M", "S", "U", "w", "W", "y", "Y", "R", "T", "%H", "%I", "%j", "%M", "%S", "%U", "%w", "%W", "%y", "%Y", "%R", "%T"`
`parse_date_time` on non-Windows supports: `"a", "A", "b", "B", "Om", "p", "r", "%a", "%A", "%b", "%B", "O%m", "%p", "%r"`

I'm not sure where to put this information :)

Reply:
Would this statement be accurate : "for `parse_date_time()` we support all formats on Linux and macOS. On windows we don't support ... "?

Reply:
@rok could you instead add a note to the binding (see https://github.com/apache/arrow/pull/14387) that explains which tokens are not supported (since it seems like most everything is)? Or make a note that punts on the whole question like the base R docs to and just say "support is platform dependent" and mostly identical to R.